topics/latency.md

```diff
@@ -159,19 +159,19 @@ Only do them if you require them, and if you are familiar with them.
 Single threaded nature of Valkey
 -------------------------------
 
-Valkey uses a *mostly* single threaded design. This means that a single process
-serves all the client requests, using a technique called **multiplexing**.
-This means that Valkey can serve a single request in every given moment, so
-all the requests are served sequentially. This is very similar to how Node.js
+Valkey uses a *mostly* single threaded design for command execution. This means that a single process
```

Comment:
Can't comment on it, but I think we should change the header to something indicates that Valkey isn't single threaded. Like:
```
Valkey sequential command execution
-----------------------------------
```

Reply:
Done